src/react-native/react-native-wizard.ts

```diff
@@ -61,43 +55,14 @@ import {
 import xcode from 'xcode';
 
 export const RN_SDK_PACKAGE = '@sentry/react-native';
-export const RN_SDK_SUPPORTED_RANGE = '>=5.0.0';
+export const RN_SDK_SUPPORTED_RANGE = '>=6.12.0';
```

Comment:
l: I think that, [If `package.json` already includes RN SDK](https://github.com/getsentry/sentry-wizard/blob/7a0443df6d87b7f2400c48bbf91481adf5de3c43/src/react-native/react-native-wizard.ts#L175) we should limit the range to `>=6.9.0` to support session replay and the feedback widget
```suggestion
export const RN_SDK_SUPPORTED_RANGE = '>=6.9.0';
```

Reply:
Resolving this comment since (obviously) the existing version check covers this. I mistakenly read `6.12.0` as ` 6.1.2` 😓